apiserver/internal/handlers/resources/resources.go

```diff
@@ -264,37 +276,38 @@ func (h *ResourceHandler) getUploadedResource(
 		resourceName:    res.Resource.Name,
 		size:            uReq.Size,
 		fingerprint:     uReq.Fingerprint,
+		pending:         pending,
 	}, nil
 }
 
 // getResourceUUID returns the resource uuid to match the new resource blob to.
```

Comment:
Please update the comment or the method name to indicate what the boolean return value represents.